docs/rules/no-import-of-scoped-modules-during-ssr.md

```diff
@@ -0,0 +1,72 @@
+# Disallow static import of scoped modules during SSR (`lwc/no-import-of-scoped-modules-during-ssr`)
+
+During server-side rendering (SSR), these modules resolve to static values on the server and are updated to actual values on the client.
+
+## Rule details
+
+**Static imports of user-scoped modules:**
```

Comment:
I may need assistance in generating the correct code for the following use cases:

1. Static import of user-scoped modules
2. Dynamic import of user-scoped modules
3. Import of a formFactor-scoped module
4. Import of a featureFlag-scoped module

If we are unable to provide the "correct" code, we might consider adding some context to explain the reasoning behind it.

Reply:
The form factor correct code is right. There is an example in the [SSR best practices doc](https://github.com/salesforce-experience-platform-emu/lwr/blob/main/packages/%40lwrjs/lwc-ssr/best_practices.md#form-factor)

Reply:
@lpomerleau  reminder that this repo is public so shouldn't link to internal urls